lisp/go-expanderr.el

```diff
@@ -25,12 +25,12 @@
           (with-current-buffer patchbuf
             (erase-buffer))
 
-	  (save-buffer)
```

Comment:
You’re not making any changes to this file, yet your editor seems to have reformatted it. Could you revert this please? We’d like to keep the git history as clean as possible :)